addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -207,6 +185,34 @@ void CreateCommands()
 
 }
 
+void CreateVIPCommands()
+{
+	// VIP Commands
+	RegAdminCmd("sm_fixbot", Admin_FixBot, g_VipFlag, "[surftimer] Toggles replay bots off and on");
+	RegAdminCmd("sm_fixbots", Admin_FixBot, g_VipFlag, "[surftimer] Toggles replay bots off and on");
+	RegAdminCmd("sm_fb", Admin_FixBot, g_VipFlag, "[surftimer] Toggles replay bots off and on");
```

Comment:
If these commands aren't regulated by the VIP convar, they shouldn't be inside CreateVIPCommands function. I don't think they should be a VIP command, perhaps an admin command in which server owners could override to a different flag if they'd like.

Reply:
Ay, glad someone else noticed this. Yeah, I didn't think they should be VIP Commands, I'll do exactly that and put them behind Admin Commands once I'm back at my computer. 

And yeah, I put them in there just to keep them all in one place, brilliant thanks for the feedback boss.

Reply:
Added in latest commit.